bundles/scripting/sightly/js-use-provider/pom.xml

```diff
@@ -70,6 +70,7 @@
                     <instructions>
                         <Sling-Bundle-Resources>/libs/sling/sightly/js;path:=/SLING-INF/libs/sling/sightly/js</Sling-Bundle-Resources>
                         <Require-Capability>io.sightly; filter:="(&amp;(version&gt;=1.0)(!(version&gt;=2.0)))"</Require-Capability>
+                        <Import-Package>org.mozilla.javascript;version="[1.1,2)",*</Import-Package>
```

Comment:
What I meant was adding a comment in the pom.xml (not in this PR), so that one understands even later why this was necessary. Not sure I completely follow you, because what you are saying is that the dependency towards  rhino (https://github.com/vladbailescu/sling/blob/0ec63ac17191b8b2ad839924d39c11e32435192e/bundles/scripting/sightly/js-use-provider/pom.xml#L174) does not generate a version in the import which is correct. But IMHO a bundle should only have bundle dependencies unless it packages the dependency into its own bundle. Therefore I would rather recommend to switch to another dependency (i.e. the bundle which exports Rhino in the correct version).